open-company/open-company-web
# WST updates

Card: https://trello.com/c/kKLd6m1M

### Needs https://github.com/open-company/open-company-auth/pull/83 but there is no need to review them together (Auth PR is also backward compatible with the 1.0 client)
### And needs https://github.com/open-company/open-company-storage/pull/198
### Which in turn needs: https://github.com/open-company/open-company-lib/pull/65

To test:
- create a new private section
- add a few users to it (but not all)
- now add a post to the new priv section
- open WST for the post
- [x] do you see only the users invited to the private section in the list?
- [x] only the users that didn't read the post have the Notify button?
- [x] do you see a Manage section green button below the chart copy?
- click it
- [x] did it open the section management panel?
- now read a post with all the users
- [x] do you see a different copy below the chart (not saying how many users have read out of how many, check the Trello card for the exact copy)
- [x] make sure the style linked in the card is implemented
- now click Notify button for an email user
- [x] do you see the copy change to `Sent to: email-of-user@domain.com`?
- now click Notify button besides a Slack user
- [ ] do you see the copy change to `Sent to: @slack-username` (if we don't have a display name for the user it will say only `Sent via Slack`?
- now add a new post with user A
- with user B don't open it but add a reaction from FoC
- refresh
- [x] post is still unread for user B?


UI checks that user has Slack, and Org w/ the bot
